pkg/internal/apply/layerApplier.go

```diff
@@ -154,6 +155,51 @@ func (a KubectlLayerApplier) decodeAddons(layer layers.Layer,
 	}
 }
 
+func (a KubectlLayerApplier) addOwnerRefs(layer layers.Layer, hrs []*helmopv1.HelmRelease) error {
+	for i, hr := range hrs {
+		a.logDebug("Adding owner ref to HelmRelease for AddonsLayer", layer, "index", i, "helmRelease", hr)
```

Comment:
be good to do `a.log.Debug` instead

Reply:
That's not how `go-logr/logr` works.  I wrote these convenience methods for myself to translate more easily, and I'm not convinced yet that I should move them out of this one gofile.

Reply:
True, you use verbosity levels. So it would be `a.Log.V(4).Info`, i really think its unnecessary to have wrapper functions like `logDebug`.